CTS_XML_TEI/perseus/pdllex/lat/ls/lat.ls.perseus-eng2.xml

```diff
@@ -33280,8 +33280,8 @@ First cut at header to Lewis and Short.
+<entryFree key="optostrotum" type="main" id="n32852a"><orth lang="la" extent="full">optostrōtum</orth>, <itype>i</itype>, <gen>n.</gen> <etym><foreign lang="greek">ὀπτόσ-στρωτόν</foreign></etym>, <sense level="1" n="I" id="n32852.3"><hi rend="ital">a brick pavement</hi>, <bibl><author>Not. Tir.</author> p. 164</bibl>.</sense></entryFree>
```

Comment:
Same as above. There is a dependent sense in the new entry that is probably now structurally inaccurate:
`<sense level="1" n="I" id="n32852.3">`

Reply:
Similar to above, the original text doesn't contain any subheadings. But in keeping with the format for the digitized version, I think this should end up as 

```
<sense level="1" n="I" id="n32852a.0">
```

Let me know what you think.

Reply:
@nkprasad12 As above.

Reply:
Done.